docs/devGuide/contributing.md

```diff
@@ -0,0 +1,72 @@
+<frontmatter>
+  title: "Contributing"
+  layout: devGuide
+  pageNav: default
+</frontmatter>
+
+# Contributing to MarkBind 
+
+Thank you for your interest in contributing! We're glad you want to help. Read the sections below to learn about our code of conduct and how to get started contributing.
+
+## Code of conduct
+
+This project and everyone participating in it are governed by our [Code of Conduct]({{baseUrl}}/devGuide/contributing/code-of-conduct.html). By participating, you are expected to uphold this code. Please report unacceptable behavior to markbind@comp.nus.edu.sg.
+
+## Things to do before getting started
+
+Make sure you have the project set up and ready. We have guides detailing what we [expect our contributors to know]({{baseurl}}/devguide/devguide#requirement), our required [environment]({{baseurl}}/devGuide.html#environment) and our [development process]({{baseurl}}/devGuide.html#development-process).
+
+## How to contribute
+
+All contributions are welcome. You can contribute to MarkBind by:
+- Reporting bugs
+- Requesting features
+- Making changes (with a corresponding Pull Request), by:
+   - Updating documentation
+   - Fixing bugs
+   - Implementing features 
+
+### How to report a bug
+
+Report bugs in our [issue tracker](https://github.com/MarkBind/markbind/issues), following the guideslines below. Following the guidelines helps us in undestanding your report better and reproduce the behavior more easily.
+
+- Has your bug been raised up before? Before reporting a bug, please search if it has been already reported in the existing issues.
+- Is your bug from the current version or an older version of MarkBind? What OS are you on? Tell us about your environment, for example, the MarkBind version you are on.
+- What did you do? What did you expect to happen? What actually happened? 
+  - Include the actual MarkBind code causing the issue, the steps for reproduction, the actual, raw output, and what you expect instead. This helps us to reproduce the bug to understand it better. 
+  - You are also encouraged to submit a PR that reproduces this in `test\functional\test_site\bugs\`.
+
+Please do not submit personal support requests as bugs, [submit a blank issue](https://github.com/MarkBind/markbind/issues/new) to our issue tracker and label it with <a href="https://github.com/MarkBind/markbind/issues?q=is%3Aopen+is%3Aissue+label%3Aquestion" class="badge" style="color:white; background-color: #A41BD6;">question</a>.
```

Comment:
`If you need help, feel free to submit a.... with a label of ...`

Reply:
Help has two meanings
1. 'I'm a contributor that there's some aspect of markbind (or its workflow) I don't understand'
2. 'I'm a user trying to make a markbind site, and I don't understand X'

I'm trying to mean the former with 'personal support requests'
Are you referring to that definition as well?

Reply:
Ahh good point, I think its naturally clear to someone reading the contributing guide that its the former though. You could add on if not.

Reply:
Oh. I mean the latter 😆

Reply:
Could be both as well, I would just leave it as it is :)